app/modules/import-export/utils.php

```diff
@@ -59,7 +59,7 @@ public static function map_old_new_term_ids( array $imported_data ) {
 	}
 
 	public static function get_elementor_post_types() {
-		return [ 'post', 'page', 'e-landing-page' ];
+		return get_post_types_by_support( 'elementor' );
```

Comment:
It will also return "elementor_library". I think it's going to be a problem.

Reply:
Have you checked it?

Reply:
Fixed